test/build_size_test.sh

```diff
@@ -11,11 +11,15 @@ set -e
 # shellcheck source=/dev/null
 source "$(dirname "${BASH_SOURCE[0]}")/../.ci/scripts/utils.sh"
 
+# TODO(https://github.com/pytorch/executorch/issues/8149): Remove -Wno-sign-compare
+# TODO(https://github.com/pytorch/executorch/issues/8357): Remove -Wno-int-in-bool-context
```

Comment:
We should have a convention for this. Since GH auto-links string like #8149, I think we could settle on `TODO(#8149): ...` when referring to an issue within the ET repo.